<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>1128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" alignment="Qt::AlignTop">
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="10" column="0">
        <widget class="QPushButton" name="reset_button">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* Set the default appearance for the push button */
    background-color: #3498db;
    border: 2px solid #3498db;
    border-radius: 5px;
    color: #fff;
    padding: 8px 12px;
    font-weight: bold;
}

QPushButton:hover {
    /* Change style when the mouse hovers over the button */
    background-color: #207dbb;
    border: 2px solid #207dbb;
}

QPushButton:pressed {
    /* Change style when the button is pressed */
    background-color: #266197;
    border: 2px solid #266197;
}</string>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="apply_check_box">
         <property name="text">
          <string>Zastosuj</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QDoubleSpinBox" name="cut_off_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox
 {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}
QDoubleSpinBox:disabled {
    color: gray;           /* Text color when disabled */
    background-color: #f0f0f0;  /* Background color when disabled */
    border: 1px solid #d4d4d4; /* Border color when disabled */
}</string>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
         </property>
         <property name="text">
          <string>Dolna częstotliwość graniczna:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
         </property>
         <property name="text">
          <string>Górna częstotliwość graniczna</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QDoubleSpinBox" name="low_cut_off">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox
 {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}
QDoubleSpinBox:disabled {
    color: gray;           /* Text color when disabled */
    background-color: #f0f0f0;  /* Background color when disabled */
    border: 1px solid #d4d4d4; /* Border color when disabled */
}</string>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
         </property>
         <property name="text">
          <string>Rodzaj filtru:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="filters_list">
         <property name="styleSheet">
          <string notr="true">QComboBox
 {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Wybierz filtr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dolnoprzepustowy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Górnoprzepustowy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pasmowo-przepustowy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pasmowo-zaporowy</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
         </property>
         <property name="text">
          <string>Częstotliwość graniczna:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QDoubleSpinBox" name="high_cut_off">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox
 {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}
QDoubleSpinBox:disabled {
    color: gray;           /* Text color when disabled */
    background-color: #f0f0f0;  /* Background color when disabled */
    border: 1px solid #d4d4d4; /* Border color when disabled */
}</string>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="dct_layout"/>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="idct_layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
